src/bci_build/package/appcollection.py

```diff
@@ -42,6 +42,12 @@ def _from_image(self) -> str | None:
 
         return f"bci/bci-base:15.{self.os_version}"
 
+    @property
+    def reference(self) -> str | None:
+        if self.os_version.is_tumbleweed:
+            return super().reference
+        return None
```

Comment:
tecnically, `{%- if image.reference %}` also works if you return an empty string, no need to change the return type of this function.

Reply:
That's true. I'm not particularly emotional about it, do you prefer this to be reverted and just use empy string for omitting?

Reply:
The only reason to use an empty string is that it would require less refactoring as you don't change the type. I leave it up to you, but an empty string might be better.